### DISCUSSION TITLE
Page Title Image

### DISCUSSION THREAD

**Discussion by christoph-noxt** [requarks/wiki#2968] [Help / Questions]
Hi!
I'm new here and excited about the possibilities Wiki.js offers. I would like to put an image in my page title on the right like in the documentation of Wiki.js on the home page. How is this possible? Many thanks in advance and best regards

**Comment by adjurk** [comment]
Welcome! :)

You can view each page's source on official docs by navigating to https://docs.requarks.io/s/home (note the /s/ prefix that redirects to page's source) for example. That way you can check how it has been rendered, since all wiki pages use Markdown.

To answer your question, simply put the following syntax at the bottom and modify the image URL and alt description:

```md
![Wiki.js](https://static.requarks.io/logo/wikijs-butterfly.svg){.align-abstopright}
```

On a side note, this behavior appears undocumented so I've created a [Pull Request](https://github.com/Requarks/wiki-docs/pull/24) to add that to the docs. Hope it gets merged soon!
